### PR TITLE
Cloudinary single upload

### PR DESCRIPTION
**Pull request by grayey**
### What this PR does (required):
- This feature uploads a photo to cloudinary.

### Screenshots / Videos (required):

https://user-images.githubusercontent.com/26135612/122002669-f942be00-cda9-11eb-98e5-3f5a3c96afd7.mp4

### Any information needed to test this feature (required):
- cd server
- npm install (there are new dependencies like the cloudinary SDK, and swagger ui)
- npm run dev

### Any issues with the current functionality (optional):
- At the moment, this only allows for single file uploads. Multiple file support is still ongoing.
 
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: client/.eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...\client\\src\\components\\ChatSideBanner\\ChatSideBanner.tsx",[],"C:\\Users\\Treal's\\Projects\\Job_tests\\hatchways\\project-shiba-new\\client\\src\\pages\\SignUp\\SignUpForm\\useStyles.ts",[],"C:\\Users\\Treal's\\Projects\\Job_tests\\hatchways\\project-shiba-new\\client\\src\\pages\\Login\\LoginForm\\useStyles.ts",[],"C:\\Users\\Treal's\\Projects\\Job_tests\\hatchways\\project-shiba-new\\client\\src\\components\\ChatSideBanner\\useStyles.ts",[],"C:\\Users\\Treal's\\Projects\\Job_tests\\hatchways\\project-shiba-new\\client\\src\\components\\AuthHeader\\useStyles.ts",[],"C:\\Users\\Treal's\\Projects\\Job_tests\\hatchways\\project-shiba-new\\client\\src\\components\\AuthMenu\\AuthMenu.tsx",[],"C:\\Users\\Treal's\\Projects\\Job_tests\\hatchways\\project-shiba-new\\client\\src\\components\\Search\\Search.tsx",[],"C:\\Users\\Treal's\\Projects\\Job_tests\\hatchways\\project-shiba-new\\client\\src\\components\\AvatarDisplay\\AvatarDisplay.tsx",[],"C:\\Users\\Treal's\\Projects\\Job_tests\\hatchways\\project-shiba-new\\client\\src\\helpers\\APICalls\\searchUsers.ts",[],"C:\\Users\\Treal's\\Projects\\Job_tests\\hatchways\\project-shiba-new\\client\\src\\components\\Search\\useStyles.ts",[]]
```
Comment (bonnieli) [inline client/.eslintcache:1]: this seems very specific to your own cache so i wouldn't want to commit this

File: server/app.js [hunk 2]
```diff
@@ -1,69 +1,90 @@
...
+const swaggerOptions = {  
+  swaggerDefinition: {  
+      info: {  
+          title:'CaninePlace',  
+          version:'1.0.0'  
+      }  
+  },  
+  apis:['routes/upload.js'],  
+} 
+const swaggerUi = require('swagger-ui-express'), swaggerJsdoc = require("swagger-jsdoc");
```
Comment (bonnieli) [inline server/app.js:29]: let's move this import up where all the rest are

File: server/models/Upload.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+const mongoose = require("mongoose");
+
+
+const uploadSchema = new mongoose.Schema({
+  file_type: {
```
Comment (bonnieli) [inline server/models/Upload.js:5]: looks like naming convention doesn't align with what @rishabdhamija has for profile models nor my suggestion 

https://github.com/hatchways/team-shiba/pull/41/files#r652276747

i would prefer it to stay consistent across the project :) a team discussion to determine that would be best

File: server/sample.env [hunk 4]
```diff
@@ -1,3 +1,4 @@
 JWT_SECRET=
-MONGO_URI=
-TEAM_NAMES=Bonnie,Sina
\ No newline at end of file
+MONGO_URI=mongodb+srv://team-shiba:team-shib%40@cluster0.s4x0p.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
```
Comment (bonnieli) [inline server/sample.env:2]: don't push up credentials

**Comment by bonnieli** [comment]
Update this branch to resolve conflicting files and we're still pending on the caching file not to be committed

**Review by bonnieli — APPROVED** [review]

**Review by rajivtitus — APPROVED** [review]
Lgtm!

Can we also resolve the conflicts and merge this branch? Thank you!

**Pull request merged**